File: dehaze-go/model/vo/image_url.go
package vo

// ImageUrlVO 图片URL视图对象
type ImageUrlVO struct {
	// ItemFileId
	ID int64 `json:"id"`
	Type string `json:"type"`
	URL string `json:"url"`
	OriginURL string `json:"originUrl"`
	Description string `json:"description"`
}

File: dehaze-android/sdk/src/main/java/com/pei/dehaze/sdk/model/dataset/ImageUrl.java
package com.pei.dehaze.sdk.model.dataset;

import lombok.Data;

/**
 * 图片URL模型类（对齐后端 ImageUrlVO）
 */
@Data
public class ImageUrl {
    private Long id;
    private String type;
    private String url;
    private String originUrl;
    private String description;
}
